### PR TITLE
Create /financial-services/resources page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Creates /financial-services/resources page
- Add secondary navigation to `financial-services` bubble

## QA

- Go to https://ubuntu-com-11672.demos.haus/financial-services/resources and compare against [copydoc](https://docs.google.com/document/d/1FIOi8l7N2On3KVVscIzWUwthhYvdkU21IhanBDUwlCI/edit#heading=h.d2yomvm8pmy4)
## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11658

### CODE REVIEW

File: templates/financial-services/resources.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
+{% extends "financial-services/_base_financial-services.html" %}
+
+{% block title %}Financial services | Resources{% endblock title %}
+{% block meta_description %}Explore a collection of useful resources that will help you understand how financial institutions can leverage secure and supported open source to accelerate their digital transformation journeys. {% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1FIOi8l7N2On3KVVscIzWUwthhYvdkU21IhanBDUwlCI/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru">
+  <div class="u-fixed-width">
```
Comment (minkyngkm) [inline templates/financial-services/resources.html:10]: I think we usually limit the width to `col-7` or something when the title is too long. 
```suggestion
  <div class="row">
    <div class="col-7">
```

File: templates/financial-services/resources.html [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Whitepapers</h2>
+  </div>
+
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-5">
+        <h3 class="p-heading--4"><a href="/engage/kubernetes-and-containerisation-for-financial-services?utm_medium=takeover&utm_campaign=7014K000000Uag0QAC">Kubernetes and Containerisation for Financial Services</a></h3>
```
Comment (minkyngkm) [inline templates/financial-services/resources.html:108]:
```suggestion
        <h3 class="p-heading--4"><a href="/engage/kubernetes-and-containerisation-for-financial-services">Kubernetes and Containerisation for Financial Services</a></h3>
```

File: templates/financial-services/resources.html [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+      </div>
+    </div>
+  </article>
+
+  <hr class="u-fixed-width is-muted">
+
+  <article class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-5">
+        <h3 class="p-heading--4"><a href="/engage/private-cloud-financial-services?utm_medium=takeover&utm_campaign=7014K000000mUlN">Private cloud for Financial Services</a></h3>
```
Comment (minkyngkm) [inline templates/financial-services/resources.html:151]:
```suggestion
        <h3 class="p-heading--4"><a href="/engage/private-cloud-financial-services">Private cloud for Financial Services</a></h3>
```

**Review by minkyngkm — COMMENTED** [review]
The code looks good to me but I will take a look again if something is updated after design review. Thanks Pete!

**Comment by petesfrench** [comment]
Thank you @minkyngkm for your review! There have been a few changes, could you take another look please?

**Comment by petesfrench** [comment]
Thanks @davegoddard42, I have made those changes. Any ideas what we can do for the 'Publications' list?

**Review by minkyngkm — APPROVED** [review]
LGTM! 🏅

**Pull request merged**